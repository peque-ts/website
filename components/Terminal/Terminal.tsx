import React from 'react';

export const Terminal: React.VFC = () => {
  return <p>Terminal</p>;
};
