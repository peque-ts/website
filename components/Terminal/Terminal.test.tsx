import { render } from '@testing-library/react';

import { Terminal } from './Terminal';

describe('Terminal', () => {
  it('should render', () => {
    const { asFragment } = render(<Terminal>npm i @pequehq/di</Terminal>);

    expect(asFragment()).toMatchSnapshot();
  });
});
