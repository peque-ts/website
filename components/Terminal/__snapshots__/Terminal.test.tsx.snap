// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Terminal should render 1`] = `
<DocumentFragment>
  <p>
    Terminal
  </p>
</DocumentFragment>
`;
